pkg/vulnsrc/vulnsrc.go

```diff
@@ -114,7 +119,9 @@ func NewUpdater(cacheDir string, light bool, interval time.Duration) Updater {
 
 func (u Updater) Update(targets []string) error {
 	log.Println("Updating vulnerability database...")
-
+	sort.Slice(targets, func(i, j int) bool {
+		return strings.Compare(targets[i], targets[j]) <= 0
+	})
```

Comment:
If we sort here, we don't have to sort in `init`.

Reply:
@knqyf263 Done